src/styles/blocks/cards.css

```diff
@@ -0,0 +1,92 @@
+
```

Comment:
this file should be names as card.css not cards.css, right? cause we have card BEM block, not cards)